omod/pom.xml

```diff
@@ -57,6 +57,12 @@
 			<type>test-jar</type>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
```

Comment:
I highly believe running `mvn clean install -DskipTests` passes without this dependency, might be the scope that you need to change